NEWS.md

```diff
@@ -237,6 +237,36 @@ Outstanding ones only.
         p C.ancestors #=> [C, M1, M2, Object, Kernel, BasicObject]
         ```
 
+    * Module#public, Module#protected and Module#private methods now accept single
+      array (or array like) argument with a list of method names.
```

Comment:
Please remove `(or array like)`. We [don't specify this in the standard documentation](https://docs.ruby-lang.org/en/master/doc/method_documentation_rdoc.html#label-Argument+Description+-28if+necessary-29)